keyboards/sam/sg81m/sg81m.h

```diff
@@ -33,5 +33,20 @@
     { K50,   K51,   K52,   K53,   K54,   K55,   K56,   K57,   K58,   K59,   K5A,   KC_NO, KC_NO, KC_NO }, \
 }
 
+#define LAYOUT_6.25u_space( \
```

Comment:
That period will cause the compiler to fail. 
```suggestion
#define LAYOUT_6_25u_space( \
```

Reply:
thanks again!